examples/tensorflow/object_detection/main.py

```diff
@@ -266,6 +267,7 @@ def evaluate(test_step, metric, test_dist_dataset, num_batches, print_freq):
     return result
 
 
+# pylint: disable=too-many-statements
```

Comment:
Please, refactor the `run` function to resolve the pylint issue.

Reply:
Thanks for the comment, refactored.